helm/kfp-operator/values.yaml

```diff
@@ -12,6 +12,7 @@ namespace:
   metadata: {}
 
 manager:
+  storedVersion: v1alpha6
```

Comment:
We decided to set the `storedVersion` in the default `values.yaml` to the current stable version (in this case, `v1alpha6`) so that if a user installs the operator from `master` using the default `values.yaml` they always get the stable version installed.

Reply:
This is now outdated because, since `manager.multiversion.enabled` is set to `false` in `helm/kfp-operator/values.yaml`, the `storedVersion` field (now moved under `manager.multiversion` won't have an effect, so we've removed it from the file entirely.

Should we keep it in even though `multiversion.enabled` is set to `false`? Just to show what's available?